### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by turtle96**
general refactoring, commenting and improving code quality

### CODE REVIEW

**Comment by turtle96** [comment]
#158

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/documentation 
pls review

**Comment by Mamtha3005** [comment]
Documentation Reviewed

File: docs/UserGuide.md [hunk 1]
```diff
@@ -169,14 +169,14 @@ Example: `hist`, then `undo 3`, will undo third command in command history.<br>
...
 
 Example: `find violin, then delete 1<br>
 This deletes the 1st task in the results of the find command.
 
 Format: `delete` INDEX<br>
  <br><img src="images/delete1.png" width="800"></br>
-Diagram 4: Deletes the first task in the list.<br>
+Diagram 6: Deletes the first task in the list.<br>
 
 <!--- @@author A0148004Rz --->
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Wrong matriculation number. There should not be a z

File: src/test/java/guitests/FindCommandTest.java [hunk 2]
```diff
@@ -39,7 +39,7 @@ public void find_byMoreThanOneTags_success() {
     /** @@author **/
 
     @Test
-    public void find_emptyList(){
+    public void find_emptyList_noResults(){
```
Comment (marcusngwenjian) [inline src/test/java/guitests/FindCommandTest.java]: Whitespace required
image

**Comment by marcusngwenjian** [comment]
Testing reviewed
Code quality reviewed

Please fix the aforementioned minor changes. 

@CS2103AUG2016-W15-C3/integration 
Ready for merging after updating.

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Pull request merged**